### ISSUE TITLE
Running compile second time fails

### ISSUE THREAD

**Issue by JCDinPGH** [rdp/ffmpeg-windows-build-helpers#600]
First, I am not a developer at all so please be gentle. I am trying to build this on Ubuntu. If I run it the first time it works fine. Running it a second time gets a message like:
From https://github.com/FFmpeg/FFmpeg
   98f87c3d29..9c590b76f4  master     -> origin/master
doing git checkout master
M       configure
M       libavcodec/Makefile
M       libavcodec/allcodecs.c
M       libavfilter/vf_frei0r.c
Already on 'master'
Your branch is behind 'origin/master' by 162 commits, and can be fast-forwarded.
  (use "git pull" to update your local branch)
Updating 3f3772e444..9c590b76f4
error: Your local changes to the following files would be overwritten by merge:
        configure
        libavcodec/Makefile
Please commit your changes or stash them before you merge.
I have tried "git pull"but it solved nothing. 
Essentially what I am hoping to do it to be able to do is build the latest ffmpeg.exe with the lastest source code. If I start from scratch, it works fine but running a second time fails. Is there something I can do to the "/ffmpeg-windows-build-helpers directory" to make it run correctly wihout having to redownload and rebuild everything from scratch?

**Comment by rdp** [comment]
I assume doing `git pull` gives the same message?  Yeah it's running into some manual patches I think.  The easiest way is --git-get-latest=y (though this is the default?)  Other way would be `cd ffmpeg...; git reset --hard; git clean -f` I think?

**Comment by JCDinPGH** [comment]
I tried cd ffmpeg...; git reset --hard; git clean -f from within the ffmpeg directory but it didn't seem to help. I can always do the "git clone https://github.com/rdp/ffmpeg-windows-build-helpers.git" and start all over. I am just trying to avoid all of the unnecessary downloads and recompiles for what hasn't changed since the previous build when it was run last.

**Comment by rdp** [comment]
You could nuke the ffmpeg_git folder.  Did you try --git-get-latest=y ?

**Comment by JCDinPGH** [comment]
> You could nuke the ffmpeg_git folder. Did you try --git-get-latest=y ?
Yes. I have been using --git-get-latest=y all along. My workaround has been to nuke the entire folder and start over. I am just trying to avoid that if possible because it makes the whole process take longer instead of just recompiling only what has changed since the previous build.

**Comment by rdp** [comment]
I think we got this figured out, if not please reopen, thanks!
